public/js/chat-cmds.js

```diff
@@ -122,13 +71,15 @@ var ListUser = function() {
         content += '<h1>User List</h1>\n';
         content += '<ul>\n';
 
-        // TODO(input): userName is not to be trusted
+        // @todo (input): userName is not to be trusted
         var peerMap = chatObj.getPeerIdToUserNameMap();
         for (var peerId in peerMap) {
-            content += itemTmpl({
-                peerId: peerId,
-                userName: peerMap[peerId]
-            });
+            if (peerMap.hasOwnProperty(peerId)) {
```

Comment:
Included a `hasOwnProperty` check in order to filter out properties on the prototype chain